src/widgets/helper/ChannelView.cpp

```diff
@@ -103,6 +104,12 @@ namespace {
                                 });
         };
 
+        // Ignore messages with this emote (useful for weeb emotes)
```

Comment:
Keep it professional
```suggestion
        // Add current emote to the list of ignored messages
```

Reply:
Added guard against multiple additions, really you shouldn't see it once it's blocked but you can still see messages sent before you blocked it; so I also added an unblock command in case you did it accidentally in the same chat.

Had a look around the codebase and couldn't see anything in `SignalVector` to remove by value, I spun up something in the lambda but if this already exists I can just change it.